### ISSUE TITLE
Bug with regard to integer and boolean output

### ISSUE THREAD

**Issue by KevinKSY** [Kvixen/Simulix#24]
The generated code has errors when you have integer and boolean output from the FMUs. The obvious errors are in the setInteger, getInteger, setBoolean, getBoolean functions. I fixed it myself, but still not working properly. Please check the bugs.

**Comment by Kvixen** [comment]
Sorry for the long wait. 

I currently have no access to Matlab and Simulink and I'm trying to get a license from my uni to keep working on Simulix. My plan is to push out a commit this week because I think I have a solution for the bugs, but I would need someone to try it out as all of my testing models I made only uses reals (Yeah, bad planing on my part). I will update this issue when I pushed the maybe-fix to a branch.
